### ISSUE TITLE
Daynamically set `limit` using js

### ISSUE THREAD

**Issue by vishu-46** [unclead/yii2-multiple-input#104]
how to set dynamic `limit` using js. show my below code

```
<?= $form->field($model, 'room_detail')->widget(MultipleInput::className(), 
            'limit' => 2,
		    'addButtonOptions' => [
			       'label' => '<i class="fa fa-plus-square"></i> Add More',
			       'class' => 'btn btn-success',
		    ]
	        'columns' => [
		        [
                    .......
                ]
             ]
]) ?>
```

**Comment by unclead** [comment]
It is not possible at this moment but you can create a PR for this feature.

**Comment by vishu-46** [comment]
how to add limit in `$wrapper` jquery.multiinput.js in function.

**Comment by unclead** [comment]
resolved via 6d612c8eeacb58bc525bf01f06dc789d93766a2b
will be available in version 2.0
